src/components/Scrollbar.tsx

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+// Used as reference: https://github.com/xobotyi/react-scrollbars-custom/issues/46#issuecomment-897506147
+
+import React, { CSSProperties, useCallback, useRef, useState, WheelEvent } from 'react'
+import Scrollbar, { ScrollbarProps } from 'react-scrollbars-custom'
+import { ElementPropsWithElementRef, ScrollState } from 'react-scrollbars-custom/dist/types/types'
+import { useTheme } from 'styled-components'
+
+import { ScrollContextProvider } from '../contexts/scroll'
+
+const paddingRight = '6px'
+const width = `calc(6px + ${paddingRight})`
+
+const paddingTop = '6px'
+const height = `calc(6px + ${paddingTop})`
+
+const createScrollbarPiece = (regular: CSSProperties, additional?: CSSProperties, useAdditionalStyling?: boolean) => ({
+  renderer: ({ elementRef, style, ...restProps }: ElementPropsWithElementRef) => (
+    <div {...restProps} ref={elementRef} style={{ ...style, ...regular, ...(useAdditionalStyling && additional) }} />
```

Comment:
Hm, how come `...(useAdditionalStyling && additional)` is not throwing any errors? If `useAdditionalStyling` is `false`, then this becomes `...false` which is not valid.

Reply:
It's perfectly valid:

```js
> { ...false }
{}
```